_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Learning common keyboard shortcuts for screen readers"
+description: "Keyboard shortcuts are hard to learn. Here are some other ways that can help that learning."
+author: anevins12
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://dequeuniversity.com/screenreaders/
+    title: "Screen Reader Keyboard Shortcuts and Gestures"
+    source: Deque University
+  - url: https://gist.github.com/anevins12/a9cc6548fcbab5f55c1dc3c7bbb0f4bd
+    title: "Source JSON file of the mindmap that can be imported into http://app.mindmapmaker.org/"
+    source: Github
+---
+
+Have you ever felt that keyboard shortcuts for assistive technology don't stick in your head? You're not alone, I've been trying to memorise them too. Keyboard shortcuts are often presented to us in a list or a table. Lists and tables are great ways of conveying information, but sometimes a more visual approach could help us learn those shortcuts.
+
+In an ideal World we would be using common screen readers daily to help us learn through experience. When that doesn't work out, here's an approach that worked for me.
+
+## Get comfortable seeing keyboard shortcuts
+
+Most keyboard shortcuts aren't fun to look at, but we need to be comfortable reading them before taking them in. 
```

Comment:
Why are they not fun?

Suggest revising to be less of an opinion statement and more of an alternate take. 

Eg “often keyboard controls are presented in a table format which some people may find dry and hard to overwhelming due to the sheer number of commands to learn”

This also does seem odd to state here, where the next paragraph then tells people to look at sites that present this information as tabular data

Reply:
I've taken that bit out and restructured a bit, focusing on the mind map a bit more. I've also taken on board the less opinionated style